bufferserver/src/test/java/com/datatorrent/bufferserver/server/ServerTest.java

```diff
@@ -70,7 +70,7 @@ public static void setupServerAndClients() throws Exception
     eventloopServer.start();
     eventloopClient.start();
 
-    instance = new Server(0, 4096,8);
+    instance = new Server(0, 4096, 8);
```

Comment:
What you say makes sense but this is an oversight where there is space after one argument and no space after another in the same line. I had originally touched this line and hence made the change.